app/code/community/Herve/CustomerOrderCancel/Helper/Data.php

```diff
@@ -37,17 +37,17 @@ public function canCancel(Mage_Sales_Model_Order $order)
             return false;
         }
 
-        // Calculate the number of days since the order's datetime
+        // Calculate the number of hours since the order's datetime
         $dateModel = Mage::getModel('core/date');
-        $createdAt = $order->getCreatedAtStoreDate();
-        $deltaDays = ($dateModel->gmtTimestamp() - $dateModel->gmtTimestamp($createdAt)) / 86400;
+        $createdAt = $order->getCreatedAt();
+        $deltaHours = ($dateModel->Timestamp() - $dateModel->Timestamp($createdAt)) / 3600;
```

Comment:
I do agree that it could be a good idea to switch from days to hours.
But why using \Mage_Core_Model_Date::timestamp instead of \Mage_Core_Model_Date::gmtTimestamp ?